cmd/ping.go

```diff
@@ -39,55 +39,74 @@ Examples:
   ping jsdelivr.com from aws+montreal --latency
 
   # Ping jsdelivr.com from a probe in ASN 123 with json output
-  ping jsdelivr.com from 123 --json`,
+  ping jsdelivr.com from 123 --json
+  
+  # Continuously ping google.com from New York
+  ping google.com from New York --infinite`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		// Create context
 		err := createContext(cmd.CalledAs(), args)
 		if err != nil {
 			return err
 		}
-
-		// Make post struct
-		opts = model.PostMeasurement{
-			Type:              "ping",
-			Target:            ctx.Target,
-			Limit:             ctx.Limit,
-			InProgressUpdates: inProgressUpdates(ctx.CI),
-			Options: &model.MeasurementOptions{
-				Packets: packets,
-			},
+		if ctx.Infinite {
+			ctx.Packets = 1
+			for {
+				ctx.From, err = ping(cmd)
+				if err != nil {
+					return err
+				}
+			}
 		}
```

Comment:
This was actually meant to work by setting packets to `16` and starting a new measurement only after that ends (starting a new measurement every time will cost more API credits, and it's also much slower), but I found there's currently a problem that we don't return parsed stats for `in-progress`... Will deal with that first and then update as to how to approach this here.

Reply:
A small update regarding this will be live soon:

We added output for packets that are likely lost, so here's an update on how to parse the output. It'll allow us to show an estimate of how many packets are lost while the measurement is still running, but isn't 100% precise. First, two examples:

1. really lost:
```
PING 1.9.9.9 (1.9.9.9) 56(84) bytes of data.
no answer yet for icmp_seq=1
no answer yet for icmp_seq=2

--- 1.9.9.9 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2084ms
```

2. not lost but slow - here the packets show as "likely lost" at first, but eventually arrive:
```
PING 35.201.25.104 (35.201.25.104) 56(84) bytes of data.
no answer yet for icmp_seq=1
64 bytes from 35.201.25.104: icmp_seq=1 ttl=56 time=278 ms
no answer yet for icmp_seq=2
64 bytes from 35.201.25.104: icmp_seq=2 ttl=56 time=278 ms
no answer yet for icmp_seq=3
64 bytes from 35.201.25.104: icmp_seq=3 ttl=56 time=279 ms

--- 35.201.25.104 ping statistics ---
4 packets transmitted, 3 received, 25% packet loss, time 608ms
rtt min/avg/max/mdev = 278.085/278.321/278.737/0.295 ms
```

1. use a regex to match `no answer yet for icmp_seq=1` and `icmp_seq=1 ttl=56 time=278 ms` parts.
2. The largest `icmp_seq` from all matches is the number of packets sent.
3. The number of `icmp_seq=1 ttl=56 time=278 ms` type matches is the number of received.
4. Calculate loss as the number of sent - the number of received.
5. At the end, update all values based on final row `4 packets transmitted, 3 received, 25% packet loss, time 608ms`

Reply:
> You can have several measurements running at the same time (because one probe already finished but the others are still running). You should call the GET on all of them in parallel, update the output once you have all responses, wait 500 ms, and repeat.
> It's realistic to expect this will result in up to five measurements running at the same time because each probe can run at different speed.

This no longer guarantees that the packets are sent in consecutive order, or I'm misunderstanding what you mean?

> Enforce max value of 5 for --limit when using --infinite

The --limit flag is supposed to be the maximum amount of measurements calls to wait for at the same time?

Reply:
> This no longer guarantees that the packets are sent in consecutive order, or I'm misunderstanding what you mean?

Yes, I don't think that'll be a problem.

Reply:
I will look at creating measurements in advance in a separate PR, the rest should work as expected here.